docs/messaging/messaging-spans.md

```diff
@@ -282,8 +288,7 @@ These attributes should be set to the broker to which the message is sent/from w
 
 - `messaging.message`: Contains attributes that describe individual messages
 - `messaging.destination`: Contains attributes that describe the logical entity
-   messages are published to and received from.
-   See [Destinations](#destinations) for more details
```

Comment:
I think those lines should stay, slightly modified?

> messages are published to. See [Destinations](#destinations) for more details

Reply:
Done d9f14c7. Sorry for the delay in this.